apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Snippets/SubscriberClientSnippets.cs

```diff
@@ -89,6 +89,56 @@ public void Overview()
         }
 
         [Fact]
+        public async Task SimpleOverview()
+        {
+            string projectId = _fixture.ProjectId;
+            string topicId = _fixture.CreateTopicId();
+            string subscriptionId = _fixture.CreateSubscriptionId();
+
+            // Sample: SimpleOverview
+            // First create a topic.
+            PublisherClient publisher = await PublisherClient.CreateAsync();
+            TopicName topicName = new TopicName(projectId, topicId);
+            publisher.CreateTopic(topicName);
+
+            // Subscribe to the topic.
+            SubscriberClient subscriber = await SubscriberClient.CreateAsync();
+            SubscriptionName subscriptionName = new SubscriptionName(projectId, subscriptionId);
+            subscriber.CreateSubscription(subscriptionName, topicName, pushConfig: null, ackDeadlineSeconds: 60);
+
+            // Publish a message to the topic using SimplePublisher.
+            SimplePublisher simplePublisher = await SimplePublisher.CreateAsync(topicName);
+            // The data can be any arbitrary ByteString.Here, we're using the string overload.
```

Comment:
Comment doesn't really make sense at the moment. Perhaps enumerate the overloads?